pyproject.toml

```diff
@@ -1,11 +1,10 @@
 [project]
 name = "imagecraft"
 description = "Create Ubuntu bootable images."
-dynamic = ["version", "readme"]
+dynamic = ["version"]
 authors = [{ name = "Canonical Ltd.", email = "snapcraft@lists.snapcraft.io" }]
-license = { file = "LICENSE" }
 dependencies = [
-    "craft-parts~=2.15",
+    "craft-parts @ git+https://github.com/canonical/craft-parts.git@work/broken-link-docs",
```

Comment:
Replace with release once https://github.com/canonical/craft-parts/pull/1182 is merged.